### DISCUSSION TITLE
Plain icon-only buttons have wrong interaction states

### DISCUSSION THREAD

**Discussion by akirichev** [primefaces/community#221] [Figma UI Kit] [answered]
I use Plain icon only button on multiple place in the prototype and on Hover/Down they are displayed as Danger buttons (in red). The same applies for the text + icon buttons. 
It seems the mapping in the Figma library is not correct. Using PrimeOne-2.1.1

**Comment by onursenture** [comment, marked as answer]
Yes, we have now checked too and found an error in the prototype links. Thank you, we have made corrections for the next release.
If you don't want to wait for the release and wish to make the corrections easily, you can follow these steps:

- Select the Button main component. From the Properties section on the right panel, open the Severity section and select all Plain variants.

<img width="1677" alt="Screenshot 2023-10-13 at 10 16 30" src="https://github.com/primefaces/community/assets/244957/d58f4db6-fc1a-4d45-80ef-840d1ee7963b">

- Switch to the Prototype tab, open the Press interaction and change the Severity section to Plain.

<img width="1676" alt="Screenshot 2023-10-13 at 10 17 22" src="https://github.com/primefaces/community/assets/244957/8e581e96-26d2-4ea6-9a92-6bff2c8a3460">

- Do the same for the Hover interaction.

<img width="1677" alt="Screenshot 2023-10-13 at 10 17 39" src="https://github.com/primefaces/community/assets/244957/35e046a3-e1e8-433f-a2aa-d9a8235723b8">

This will fix all the plain button prototypes.

**Reply by akirichev** [reply]
Thanks for the detailed explanation. I already did the fix on my end because I expected the fix to take longer.

**Reply by heath-capstone** [reply]
Hello @onursenture. It appears that this issue has been fixed in the 2.2.0 release; can you confirm that for me please? I just downloaded 2.2.0 and am looking to get started. Thanks!

**Reply by onursenture** [reply]
Yes @heath-capstone, we solved this with 2.2.0.
